src/code_F5750.c

```diff
@@ -238,7 +238,9 @@ INCLUDE_ASM(s32, "code_F5750", func_802D2484);
 
 INCLUDE_ASM(s32, "code_F5750", func_802D249C);
 
-INCLUDE_ASM(s32, "code_F5750", func_802D24F4);
+ApiStatus func_802D24F4(ScriptInstance* script, s32 isInitialCall) {
+    return (gPlayerStatus.moveFrames == 0) * ApiStatus_DONE2;
```

Comment:
Sigh. Clover thought that they added DONE1 specifically to allow returning a bool, but I guess that's not the case. Maybe we'll find out what DONE1 does soon!

Reply:
I don't see how this negates done1 from being a bool. In this case, they're deciding if they should block or not based *on* a bool value, but aren't attempting to return a bool in particular, so I think that's still on the table?